### PR TITLE
APEXCORE-460 A keytab passed from command line will be used for token refresh

### PR DESCRIPTION
**Pull request by pramodin**
@davidyan74 @gauravgopi123 please see

### CODE REVIEW

File: engine/src/main/java/com/datatorrent/stram/client/StramAppLauncher.java [hunk 1]
```diff
@@ -577,17 +604,9 @@ public ApplicationId launchApp(AppFactory appConfig) throws Exception
...
-      try (FileSystem fs = StramClientUtils.newFileSystemInstance(conf)) {
-        Path destPath = new Path(StramClientUtils.getDTDFSRootDir(fs, conf), localKeyTabPath.getName());
-        if (!fs.exists(destPath)) {
-          fs.copyFromLocalFile(false, false, localKeyTabPath, destPath);
-        }
-        dag.setAttribute(LogicalPlan.KEY_TAB_FILE, destPath.toString());
-      }
+    // TODO:- Need to see if other token refresh attributes are needed if security is not enabled
+    if (UserGroupInformation.isSecurityEnabled()) {
+      setTokenRefreshKeytab(dag, conf);
```
Comment (gauravgopi123) [inline engine/src/main/java/com/datatorrent/stram/client/StramAppLauncher.java:609]: Is this refactoring required? This function is not used any where else too....
I don't see any change in the existing code and refactored code other than one new log message at warn level

Comment (gauravgopi123) [inline engine/src/main/java/com/datatorrent/stram/client/StramAppLauncher.java:609]: I see that setTokenRefreshKeytab method is used in testing..

Looks good to me 👍

Comment (pramodin) [inline engine/src/main/java/com/datatorrent/stram/client/StramAppLauncher.java:609]: Yes @gauravgopi123 that way it can be tested

Comment (pramodin) [inline engine/src/main/java/com/datatorrent/stram/client/StramAppLauncher.java:609]: Also change is that it checks if the command line keytab from StramUserLogin.getKeytab() is available

**Pull request merged**